charts/daemonset/README.md

```diff
@@ -56,15 +56,22 @@ helm delete daemonset
 | image.tag | string | `"latest"` |  |
 | image.isRepositoryPublic | bool | `true` | Whether the repository is public |
 | image.pullPolicy | string | `"IfNotPresent"` | The image pullPolicy to use |
+| annotations | object | `{}` | Annotations to be applied to the daemonset |
 | config.command | list | `[]` |  |
 | config.args | list | `[]` |  |
 | config.env | string | `nil` | Map of environment variables to use within the job |
 | config.secrets | object | `{}` | Map of secrets that will be exposed as environment variables within the job |
+| config.securityContext | object | `{}` | Map of securityContext object applied to pods |
+| config.hostPID | bool | `false` | Allow pods access to host process information |
```

Comment:
Maybe if this access related appending "access" to the name of the input gives some better context on what the setting does?  For me when I read `config.hostPID` I was thinking I needed to enter in a PID.